test/extended/image_ecosystem/s2i_perl.go

```diff
@@ -18,15 +19,15 @@ import (
 var _ = g.Describe("[image_ecosystem][perl][Slow] hot deploy for openshift perl image", func() {
 	defer g.GinkgoRecover()
 	var (
-		dancerTemplate = "https://raw.githubusercontent.com/openshift/dancer-ex/master/openshift/templates/dancer-mysql.json"
-		oc             = exutil.NewCLI("s2i-perl", exutil.KubeConfigPath())
-		modifyCommand  = []string{"sed", "-ie", `s/data => \$data\[0\]/data => "1337"/`, "lib/default.pm"}
-		pageCountFn    = func(count int) string { return fmt.Sprintf(`<span class="code" id="count-value">%d</span>`, count) }
-		dcName         = "dancer-mysql-example"
-		rcNameOne      = fmt.Sprintf("%s-1", dcName)
-		rcNameTwo      = fmt.Sprintf("%s-2", dcName)
-		dcLabelOne     = exutil.ParseLabelsOrDie(fmt.Sprintf("deployment=%s", rcNameOne))
-		dcLabelTwo     = exutil.ParseLabelsOrDie(fmt.Sprintf("deployment=%s", rcNameTwo))
+		appSource     = exutil.FixturePath("testdata", "image_ecosystem", "perl-hotdeploy")
+		perlTemplate  = exutil.FixturePath("testdata", "image_ecosystem", "perl-hotdeploy", "perl.json")
```

Comment:
Could maybe use appSource here instead of the full path, but this is probably easier to decipher

Reply:
yeah i'm gonna leave this one.  I didn't want to deal w/ additional path concatenation in a separate call.